ably/util/exceptions.py

```diff
@@ -61,6 +62,17 @@ def raise_for_response(response):
                             status_code=response.status_code,
                             code=response.status_code * 100)
 
+    @staticmethod
+    def decode_error_response(response):
+        content_type = response.headers.get('content-type')
+        if isinstance(content_type, str):
+            if content_type.startswith('application/x-msgpack'):
+                return msgpack.unpackb(response.content)
```

Comment:
This is not necessary since msgpack-python v1, which this library depends on, uses `raw=False` by default, see [this resolved issue](https://github.com/msgpack/msgpack-python/issues/99).